we.dev.config.example.ts

```diff
@@ -86,12 +86,12 @@ export default defineConfig({
       subtitle: 'Collaborative note taking',
       description: 'Real-time notetaking based on syn',
       icon: {
-        type: 'https',
-        url: 'https://lightningrodlabs.org/projects/notebooks.png',
+        type: 'filesystem',
```

Comment:
Oh, and can you undo those changes to the example config? The example config is intended to work out of the box when cloning the repository so it should not contain references to local paths. For developing on Moss I use a dedicated `we.dev.config.ts` file which is used by the `yarn applet-dev` and `yarn applet-dev-1` commands and which is in the `.gitignore` file.

Reply:
whoops that was a mistake.  Ah, that's a good idea.